README.rst

```diff
@@ -60,6 +60,14 @@ can use the :code:`@pytest.mark.repeat(count)` decorator:
    def test_repeat_decorator():
        pass
 
+If you want to override default tests executions order, you can use :code:`--repeat-scope`
+command line option with one of the next values: :code:`session`,  :code:`module`, :code:`class` of :code:`function`(default).
+It behaves like a scope of the py.test fixture.
+
+:code:`function` (default) scope repeats each test :code:`count` or :code:`repeat` times before executing next test.
+:code:`session` scope repeats whole tests session, i.e. all collected tests executed once, then all such tests executed again and etc.
+:code:`class` and :code: `module` behaves similar, but repeating tests block is all tests for class or module.
```

Comment:
`module` is not formatted, likely due to the space after ":code:"

Reply:
It's not clear what you mean by "repeating tests block is all tests for class or module". Could you rephrase?